Cargo.toml

```diff
@@ -13,7 +13,10 @@ members = [
 ]
 # re-add when all crates are MSRV 1.60+
 exclude = [
+    "aead",
+    "async-signature",
     "crypto",
+    "signature",
     "elliptic-curve"
```

Comment:
You can remove `crypto` and `elliptic-curve` from this and add them to `members`. Likewise `aead` et al need to be removed from `members`.

I think you also need `universal-hash` in `exclude`?